src/app/main.py

```diff
@@ -3,7 +3,7 @@
 from fastapi.openapi.utils import get_openapi
 
 from app import config as cfg
-from app.api.routes import login, users, sites, events, devices, media, installations, alerts
+from app.api.routes import login, users, sites, events, devices, media, installations, alerts, access
```

Comment:
Quick (annoying) thoughts: all the routes have plural names, should we make is "accesses" for consistency?

Reply:
I thought about it but thought it was weird naming. But will fix you are right